Pipfile

```diff
@@ -22,7 +22,7 @@ django-graphql-jwt = ">=0.1.13"
 django-impersonate = ">=1.3"
 django-measurement = "==3.0.0"
 django-mptt = ">=0.7.1"
-django-phonenumber-field = ">=1.3.0,<2.4.0"
+django-phonenumber-field = ">=1.3.0,<3.0"
```

Comment:
May there is no need to restrain it to ``<3.0``` since we prefer the latest version?

Reply:
Besides, don't forget to update the requiremets files. :D

Reply:
I don't know, if they do release a 3.0, it would definitively break. So I don't know if we should make the screw thigh or not.

Thanks, forgot about that!